04_numeronym/CameronSchafer/main_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// main() test
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := "[a11y K8s abc]\n"
+	actual := buf.String()
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
+}
+
+// numeronyms()
+func TestNumeronyms(t *testing.T) {
+	// test cases with descriptions.
+	testCases := []struct {
+		description string
+		input       []string
+		expectedArr []string
+	}{
+		{description: "numeronyms('accessibility', 'Kubernetes', 'abc')",
```

Comment:
Hi, please consider the following scenarios for your test cases and reimplement your functions. 

- When if the input isn't a word (e.g. !international), should this be processed? What if the string is "12345"? Should the output be "135"?
- What happens if the string has two words? e.g. "Web Development"? (Wiki page on numeronyms has some clues)
- How do you handle non-alphanumerical characters?